sys/posix/include/arpa/inet.h

```diff
@@ -0,0 +1,259 @@
+/*
+ * Copyright (C) 2013-15 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @addtogroup  pnet
+ * @{
+ */
+
+/**
+ * @file
+ * @brief   Definitions for internet operations
+ * @see     <a href="http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/arpa_inet.h.html">
+ *              The Open Group Base Specifications Issue 7, <arpa/inet.h>
+ *          </a>
+ *
+ * @todo    Implement inet_addr() and inet_ntoa() (require some form of allocation)
+ *
+ * @author  Martine Lenders <mlenders@inf.fu-berlin.de>
+ */
+#ifndef ARPA_INET_H
+#define ARPA_INET_H
+
+#include <inttypes.h>
+#include <stdio.h>
+
+#include "byteorder.h"
+#include "net/ipv4/addr.h"
+#include "net/ipv6/addr.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef uint16_t in_port_t;         /**< Internet port type */
+typedef uint32_t in_addr_t;         /**< IPv4 address type */
```

Comment:
no, they aren't, they are just equivalent to the `uint16_t` and `uint32_t` defined in `<inttypes.h> [1]. (:coffee: + :eyeglasses:)

[1] http://pubs.opengroup.org/onlinepubs/000095399/basedefs/netinet/in.h.html